load_testing/lib/flow_sp_ial2_sign_up_async.py

```diff
@@ -114,6 +114,41 @@ def ial2_sign_up_async(context):
         },
     )
 
+    auth_token = authenticity_token(resp)
+
+    resp = do_request(
+        context,
+        "post",
+        "/authentication_methods_setup",
+        "/phone_setup",
+        "",
+        {
+            "_method": "patch",
+            "two_factor_options_form[selection][]": "",
+            "two_factor_options_form[selection][]": "phone",
```

Comment:
```suggestion
            "two_factor_options_form[selection][]": "phone",
```
I just confirmed via REPL that the second key overwrites the first